sequencer/config/config.go

```diff
@@ -22,34 +20,17 @@ type Duration struct {
 // ForgeBatchGasCost is the costs associated to a ForgeBatch transaction, split
 // into different parts to be used in a formula.
 type ForgeBatchGasCost struct {
-	Fixed     uint64 `validate:"required" env:"TONNODE_FORGEBATCHGASCOST_FIXED"`
-	L1UserTx  uint64 `validate:"required" env:"TONNODE_FORGEBATCHGASCOST_L1USERTX"`
-	L1CoordTx uint64 `validate:"required" env:"TONNODE_FORGEBATCHGASCOST_L1COORDTX"`
-	L2Tx      uint64 `validate:"required" env:"TONNODE_FORGEBATCHGASCOST_L2TX"`
-}
-
-// CoordinatorAPI specifies the configuration parameters of the API in mode
-// coordinator
-type CoordinatorAPI struct {
-	// Coordinator enables the coordinator API endpoints
-	Coordinator bool `env:"TONNODE_COORDINATORAPI_COORDINATOR"`
+	Fixed    uint64 `validate:"required" env:"TONNODE_FORGEBATCHGASCOST_FIXED"`
+	L1UserTx uint64 `validate:"required" env:"TONNODE_FORGEBATCHGASCOST_L1USERTX"`
```

Comment:
@Intizar-T Did you declare these env variables, if so how did you find the exact number for gas cost?